Docker/Container.php

```diff
@@ -258,6 +268,11 @@ function ($event) use ($containerName, &$containerId) {
                 if ($event['host'] != substr($containerId, 0, strlen($event['host']))) {
                     $this->logger->notice("Invalid container host " . $event['host'], $event);
                 } else {
+                    array_walk_recursive($event, function (&$value) {
+                        if (is_scalar($value)) {
```

Comment:
je to tady potřeba?

>  Any key that holds an array will not be passed to the function.

http://php.net/manual/en/function.array-walk-recursive.php

každopádně zajímavý použití, tohle by mě vůbec nenapadlo a šel bych na to přes `array_map`

Reply:
array_map neni rekurzivni, ne? s is_scalar dobrej postreh! :)

Reply:
není, musel by se zanořit sám do sebe :-)